features/admin.api-resources.v1/components/api-resource-panes/index.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
No need to update the license headers of unchanged files. Check other places as well.